### DISCUSSION TITLE
Airplay to MA and MA to Forward to Speakers

### DISCUSSION THREAD

**Discussion by User873902** [music-assistant/support#2729] [Feature requests and ideas]
Requesting the ability to AirPlay music, podcasts, or videos to Music Assistant (as a target) for Music Assistant to forward the audio to the speaker groups. Perhaps this feature seems a little silly, but rather than using standard supported smart speaker protocol, I would like to use the native iPhone protocol to play the audio to otherwise non-supported speakers and, more importantly, speaker groups.

**Comment by OzGav** [comment]
duplicate https://github.com/orgs/music-assistant/discussions/1757
